misc/pkg_install.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+LIB_7="vim build-essential ccache libgc-dev graphviz libunwind7-dev pkg-config"
+LIB_8="vim build-essential ccache libgc-dev graphviz libunwind8-dev pkg-config"
+MSG="The following package will be installed [ vim | build-essential | ccache | libgc-dev | graphviz | libunwind-dev | pkg-config ]"
+MSG_VIM="Do you want to set your vim for NIT ? [y/n] "
+
+. /etc/os-release
+BOOL=0
+
+function vim_conf {
+	NIT_PATH=$PWD
+	sudo mkdir -p ~/.vim/autoload ~/.vim/bundle
+	echo ".Vim directory created [OK]"
+
+	# Check if crul is instaled or not
+	CURL=$(dpkg-query -W -f='${Status}\n' "curl")
+	if [ "$CURL" = "install ok installed" ]; then
+		curl -LSso ~/.vim/autoload/pathogen.vim https://tpo.pe/pathogen.vim
+	else
+		echo "Curl instalation ...."
+		apt-get install curl
+		curl -LSso ~/.vim/autoload/pathogen.vim https://tpo.pe/pathogen.vim
+	fi
+
+	echo "call pathogen#infect()" >> /etc/vim/vimrc
+	echo "syntax on" >> /etc/vim/vimrc
+	echo "filetype plugin indent on" >> /etc/vim/vimrc
+	echo "Pathogen installation [OK]"
```

Comment:
We do not want to mess the global etc file. the local one is enough (~/.vimrc)
Also, if it already exists or is already configured, we do not want to mess with it.
In fact, running the script a second time should be a no-op